changelog/6327.deprecation.rst

```diff
@@ -1,3 +1,3 @@
-Using `sunpy.timeseries.GenericTimeSeries.index` is deprecated.
```

Comment:
The filename is wrong here.

Reply:
This is an old changelog we didnt remove?!